File: ComponentImpl/src/main/java/com/xiaojinzi/component/error/ignore/TargetActivityNotFoundException.java
package com.xiaojinzi.component.error.ignore;

/**
 * 表示目标界面没有找到
 *
 * time   : 2018/11/11
 *
 * @author : xiaojinzi
 */
public class TargetActivityNotFoundException extends RuntimeException {

    public TargetActivityNotFoundException() {
    }

    public TargetActivityNotFoundException(String message) {
        super(message);
    }

    public TargetActivityNotFoundException(String message, Throwable cause) {
        super(message, cause);
    }

    public TargetActivityNotFoundException(Throwable cause) {
        super(cause);
    }

}

File: ComponentImpl/src/main/java/com/xiaojinzi/component/error/ignore/TargetActivityNotFoundException.kt
package com.xiaojinzi.component.error.ignore

import java.lang.RuntimeException

/**
 * 表示目标界面没有找到
 *
 * time   : 2018/11/11
 *
 * @author : xiaojinzi
 */
class TargetActivityNotFoundException(message: String? = null) : RuntimeException(message)